app/src/main/java/at/roteskreuz/stopcorona/model/workers/SelfRetestNotifierWorker.kt

```diff
@@ -31,7 +31,7 @@ class SelfRetestNotifierWorker(
 
             workManager.enqueueUniquePeriodicWork(
                 TAG,
-                ExistingPeriodicWorkPolicy.REPLACE,
+                ExistingPeriodicWorkPolicy.KEEP,
```

Comment:
This is not probably wanted behaviour. We have a limit for 100 jobs at the time and once quarantine is updated, we are not interested anymore of the old schedules.

Reply:
```
    /**
     * If there is existing pending (uncompleted) work with the same unique name, do nothing.
     * Otherwise, insert the newly-specified work.
     */
    KEEP
```

Reply:
The idea is to keep the old periodic schedule instead of setting a new one.

Reply:
👍